e2e/specs/create-form-using-custom-schema.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { test } from "../core";
+import { expect } from "@playwright/test";
+import { deleteForm } from "../commands/form-operations";
+import { FormBuilderPage } from "../pages";
+import customSchema from "../support/custom-schema.json";
+
+let formUuid = null;
+
+test("Create a form using a custom JSON schema", async ({ page }) => {
+  const formBuilderPage = new FormBuilderPage(page);
+
+  await test.step("When I visit the form builder", async () => {
+    await formBuilderPage.gotoFormBuilder();
+  });
+
+  await test.step("And click the `Create New Form` button", async () => {
+    await formBuilderPage.createNewFormButton().click();
+  });
+
+  await test.step("Then I paste in a custom JSON schema into the schema editor and click the `Render Changes` button", async () => {
+    await formBuilderPage.schemaInput().fill(JSON.stringify(customSchema));
+    // Inputs the custom schema and render changes
+    await formBuilderPage.renderChangesButton().click();
+  });
+
+  await test.step("Then I click the `Save Form` button", async () => {
+    await formBuilderPage.saveForm();
+  });
+
+  await test.step(" I should get a success message and be redirected to the edit page for the new form", async () => {
```

Comment:
```suggestion
  await test.step("And I should get a success message and be redirected to the edit page for the new form", async () => {
```

steps should start with either one of the following term:

- Given
- When
- Then
- And
- But

I'll update the guide

Reply:
ex: https://docs.behat.org/en/v2.5/guides/1.gherkin.html